src/condor_negotiator.V6/matchmaker.cpp

```diff
@@ -4872,13 +5060,39 @@ matchmakingAlgorithm(const char *scheddName, const char *scheddAddr, ClassAd &re
 	bool			newBestFound;
 		// to store results of evaluations
 	string remoteUser;
+	string requestUser;
 	classad::Value	result;
 	bool			val;
 		// request attributes
 	int				requestAutoCluster = -1;
 
 	dprintf(D_FULLDEBUG, "matchmakingAlgorithm: limit %f used %f pieLeft %f\n", submitterLimit, limitUsed, pieLeft);
 
+    // initialize group usage cache (for non-leaf groups)
+    populateGroupUsageCache();
```

Comment:
There is a segmentation fault here when running with traditional accounting (no groups).

Caught signal 11: si_code=1, si_pid=4, si_uid=0, si_addr=0x4
Stack dump for process 29617 at timestamp 1589789674 (13 frames)
/usr/sbin/../lib/libcondor_utils_8_6_13.so(dprintf_dump_stack+0x24)[0x7f1d6706d334]
/usr/sbin/../lib/libcondor_utils_8_6_13.so(_Z17unix_sig_coredumpiP9siginfo_tPv+0x6a)[0x7f1d671ba32a]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x12890)[0x7f1d65359890]
condor_negotiator(_ZNK9HashTableI8MyStringfE6lookupERKS0_Rf+0x0)[0x55dbc418e630]
condor_negotiator(_ZN10Matchmaker20matchmakingAlgorithmEPKcS1_RN14compat_classad7ClassAdERNS2_27ClassAdListDoesNotDeleteAdsEddddddb+0x24b)[0x55dbc4182e3b]
condor_negotiator(_ZN10Matchmaker9negotiateEPKcS1_PKN14compat_classad7ClassAdEdddRNS2_27ClassAdListDoesNotDeleteAdsERSt3mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESt3setISE_St4lessISE_ESaISE_EESH_SaISt4pairIKSE_SJ_EEEblRiRd+0x582)[0x55dbc4189a52]
condor_negotiator(_ZN10Matchmaker18negotiateWithGroupEiddRN14compat_classad27ClassAdListDoesNotDeleteAdsERSt3mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESt3setIS9_St4lessIS9_ESaIS9_EESC_SaISt4pairIKS9_SE_EEES2_fPKc+0xf8c)[0x55dbc418b91c]
condor_negotiator(_ZN10Matchmaker15negotiationTimeEv+0x19ca)[0x55dbc418d87a]
/usr/sbin/../lib/libcondor_utils_8_6_13.so(_ZN12TimerManager7TimeoutEPiPd+0x3b7)[0x7f1d671c83c7]
/usr/sbin/../lib/libcondor_utils_8_6_13.so(_ZN10DaemonCore6DriverEv+0x686)[0x7f1d671acf66]
/usr/sbin/../lib/libcondor_utils_8_6_13.so(_Z7dc_mainiPPc+0x1625)[0x7f1d671bdd05]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xe7)[0x7f1d64f77b97]
condor_negotiator(_start+0x2a)[0x55dbc416306a]